pkgs/applications/office/scribus/default.nix

```diff
@@ -1,8 +1,9 @@
 { stdenv, fetchurl, pkgconfig, freetype, lcms, libtiff, libxml2
 , libart_lgpl, qt4, python2, cups, fontconfig, libjpeg
-, zlib, libpng, xorg, cairo, podofo, aspell, boost, cmake }:
+, zlib, libpng, xorg, cairo, podofo, aspell, boost, cmake, imagemagick }:
 
 let
+  icon = ./icon.svg;
```

Comment:
Can we download this from somewhere? Ideally it would be included in the the src

Reply:
No, the original SVG file is [here](http://linux.hanski.info/static/scribus_promotion/scribus-icon2.svg), but it is not square nor centered so I’ve edited it. If you know how we can script that on the SVG file or when converting it to PNG we are good, but else we need to use this custom file.

Reply:
How about supplying icon.svg as a gist? This way it would not pollute the Nixpkgs source.

Reply:
@erictapen Nice idea, I’ll do this.

Reply:
Done. I can rebase and squash the commit if needed.